### ISSUE TITLE
reword `#4: "west":500 ESTABLISHED_CHILD_SA (established Child SA); REKEY in XXs; REPLACE in XXs; newest; ...`

### ISSUE THREAD

**Issue by cagney** [libreswan/libreswan#1670]
in the `ipsec status` output,  something like:
```
#4: "west":500 state: established Child SA; REKEY in XXs; ...
```
perhaps

**Comment by fatma-hashem** [comment]
I would like to work on this issue. Can you assign it to me?

**Comment by cagney** [comment]
good choice

This is from `testing/pluto/ikev2-20-ikesa-reauth`
```
 ipsec status | grep "ESTABLISHED_"
#3: "west":500 ESTABLISHED_IKE_SA (established IKE SA); REPLACE in XXs; newest; idle;
#4: "west":500 ESTABLISHED_CHILD_SA (established Child SA); REKEY in XXs; REPLACE in XXs; newest; eroute owner; IKE SA #3; idle;
```
`ESTABLISHED_IKE_SA` is an internal detail, and, hence what should be removed, the result would be something like:
```
#3: "west":500 established IKE SA; REPLACE in XXs; newest; idle;
#4: "west":500 established Child SA; REKEY in XXs; REPLACE in XXs; newest; eroute owner; IKE SA #3; idle;
```
but notice how the test greps for `ESTABLISHED`; things like that will need updating in the testsulte.

**Comment by fatma-hashem** [comment]
@cagney Ive submitted a PR https://github.com/libreswan/libreswan/pull/2645
Please let me know if any further adjustment is needed!

**Comment by The-Mule** [comment]
I am all in for making output more compact. On the other hand, any change in `ipsec status` might be breaking some automation based on that (since we don't have "machine readable" status yet). For instance, openvswitch is parsing ipsec status output for monitoring and correction purposes (this will break neither old nor current openvswitch use case IMO (cc @igsilya).

**Comment by igsilya** [comment]
FWIW, OVS only relies on the format of the lines like these:
```
"west": 0.0.0.0/0===192.1.2.45[@west]...192.1.2.23[@east]===0.0.0.0/0; routed-tunnel; my_ip=unset; their_ip=unset;

#2: "west" esp.ESPSPIi@192.1.2.23 esp.ESPSPIi@192.1.2.45 tun.0@192.1.2.23 tun.0@192.1.2.45 Traffic: ESPin=84B ESPout=84B ESPmax=2^63B 
```
And they are not touched by the change, so should be fine.

**Comment by cagney** [comment]
@The-Mule, yes `ipsec status` can be a bit of an untouchable; I guess #1553 might fall into that category.

The above is the log generated when a connection transitions to established.  `ESTABLISHED_CHILD_SA` is an internal enum  (it's changed a few times in the past) that clients hopefully don't rely on.
